GamaGameHub/GamaGameHub/Controllers/HomeController.cs

```diff
@@ -21,9 +24,30 @@ public IActionResult AboutUs()
             return View();
         }
 
+        [HttpGet]
         public IActionResult ContactUs()
         {
             return View();
         }
+
+        [HttpPost]
+        public IActionResult ContactUs(ContactUsViewModel model)
+        {
+            if (!ModelState.IsValid) { return View(model); }
+
+            MailMessage mailMessage = new MailMessage(model.From,"meribelcheva@gmail.com");
+            mailMessage.Subject = model.Subject;
+            mailMessage.Body = $"Name: {model.SenderName}\nEmail: {model.From}\nMessage: {model.Body}\n";
+            mailMessage.IsBodyHtml = false;
+
+            SmtpClient smtpClient = new SmtpClient();
+            smtpClient.Host = "smtp.gmail.com";
+            smtpClient.Port = 587;
+            smtpClient.EnableSsl = true;
+            smtpClient.Credentials = new NetworkCredential("meribelcheva@gmail.com", "beky gwha odgp hbnj");
```

Comment:
Credentials should not be directly inside the code, somewhere in a different file, i.e. and environment variable file, or simply in a const string above the code etc.